mne/tests/test_source_estimate.py

```diff
@@ -543,125 +490,6 @@ def test_extract_label_time_course():
-    pytest.raises(ValueError, stc_from.morph_precomputed,
-                  subject_to, [vertices_to[0][:-1], vertices_to[1]], morph_mat)
-    pytest.raises(ValueError, stc_from.morph_precomputed, subject_to,
-                  vertices_to, morph_mat, subject_from='foo')
-
-    # steps warning
-    with warnings.catch_warnings(record=True) as w:
-        warnings.simplefilter('always')
-        compute_morph_matrix(subject_from, subject_to,
-                             stc_from.vertices, vertices_to,
-                             smooth=1, subjects_dir=subjects_dir)
-    assert_equal(len(w), 2)
-
-    mean_from = stc_from.data.mean(axis=0)
-    mean_to = stc_to1.data.mean(axis=0)
-    assert (np.corrcoef(mean_to, mean_from).min() > 0.999)
-
-    # make sure we can fill by morphing
-    stc_to5 = morph_data(subject_from, subject_to, stc_from, grade=None,
-                         smooth=12, buffer_size=3, subjects_dir=subjects_dir)
-    assert (stc_to5.data.shape[0] == 163842 + 163842)
-
-    # Morph sparse data
-    # Make a sparse stc
-    stc_from.vertices[0] = stc_from.vertices[0][[100, 500]]
-    stc_from.vertices[1] = stc_from.vertices[1][[200]]
-    stc_from._data = stc_from._data[:3]
-
-    pytest.raises(RuntimeError, stc_from.morph, subject_to, sparse=True,
-                  grade=5, subjects_dir=subjects_dir)
-
-    stc_to_sparse = stc_from.morph(subject_to, grade=None, sparse=True,
-                                   subjects_dir=subjects_dir)
-    assert_array_almost_equal(np.sort(stc_from.data.sum(axis=1)),
-                              np.sort(stc_to_sparse.data.sum(axis=1)))
-    assert_equal(len(stc_from.rh_vertno), len(stc_to_sparse.rh_vertno))
-    assert_equal(len(stc_from.lh_vertno), len(stc_to_sparse.lh_vertno))
-    assert_equal(stc_to_sparse.subject, subject_to)
-    assert_equal(stc_from.tmin, stc_from.tmin)
-    assert_equal(stc_from.tstep, stc_from.tstep)
-
-    stc_from.vertices[0] = np.array([], dtype=np.int64)
-    stc_from._data = stc_from._data[:1]
-
-    stc_to_sparse = stc_from.morph(subject_to, grade=None, sparse=True,
-                                   subjects_dir=subjects_dir)
-    assert_array_almost_equal(np.sort(stc_from.data.sum(axis=1)),
-                              np.sort(stc_to_sparse.data.sum(axis=1)))
-    assert_equal(len(stc_from.rh_vertno), len(stc_to_sparse.rh_vertno))
-    assert_equal(len(stc_from.lh_vertno), len(stc_to_sparse.lh_vertno))
-    assert_equal(stc_to_sparse.subject, subject_to)
-    assert_equal(stc_from.tmin, stc_from.tmin)
-    assert_equal(stc_from.tstep, stc_from.tstep)
-
-    # Morph vector data
-    stc_vec = _real_vec_stc()
-
-    # Ignore warnings about number of steps
-    stc_vec_to1 = stc_vec.morph(subject_to, grade=3, smooth=12,
-                                buffer_size=1000, subjects_dir=subjects_dir)
-    stc_vec_to2 = stc_vec.morph_precomputed(subject_to, vertices_to, morph_mat)
-    assert_array_almost_equal(stc_vec_to1.data, stc_vec_to2.data)
```

Comment:
same remark. I would put it back.

Reply:
unless you confirm that you just copy pasted into test_morph.py

Reply:
I mean I copy pasted and removed the buffer size, set the asserts to the new default...